### PR TITLE
test(di): test local registrations

### PR DESCRIPTION
**Pull request by fkleuver**
# Pull Request

## 📖 Description

@jwx I can't seem to reproduce the problem you had with locally registered deps. Can you help me out here?

## 👩‍💻 Reviewer Notes

Just a couple of DI local dependency tests to get to the bottom of a router related issue

### CODE REVIEW

**Comment by fkleuver** [comment]
The underlying issue is solved now but this is still a relevant test file to have, so let's get this in @EisenbergEffect ?

**Comment by fkleuver** [comment]
@BBosman Any remarks on this PR from a typing/linting perspective? Trying to make sure new tests conform now.

File: packages/jit-html/test/integration/di-registrations.spec.ts [hunk 1]
```diff
@@ -0,0 +1,328 @@
+import { TestContext } from '../util';
```
Comment (BBosman) [inline packages/jit-html/test/integration/di-registrations.spec.ts]: These aren't alphabetized correctly

File: packages/jit-html/test/integration/di-registrations.spec.ts [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+import { CustomElementResource, Aurelia, ICustomElement, IDOM, IProjectorLocator, IRenderingEngine, INode, LifecycleFlags, Scope, BindingContext } from '@aurelia/runtime';
+import { expect } from 'chai';
+import { InterfaceSymbol } from '@aurelia/kernel';
+
+
```
Comment (BBosman) [inline packages/jit-html/test/integration/di-registrations.spec.ts]: Bonus newline

**Comment by BBosman** [comment]
@fkleuver Just a few nits. Looking good otherwise.

**Pull request merged**